peerdid/peer_did.py

```diff
@@ -51,7 +86,14 @@ def resolve_peer_did(peer_did: PEER_DID, version_id=None) -> JSON:
     version_id is not used for now, as we support only static layer where did doc never changes
     :return: resolved did_doc as a JSON string
     """
-    pass
+    if is_peer_did(peer_did):
+        if peer_did[9] == '0':
+            inception_key = peer_did[9:]
+            pass
+        if peer_did[9] == '2':
+            pass
+    else:
+        raise ValueError('Wrong Peer DID')
 
 
 def save_peer_did(peer_did: PEER_DID, storage: Optional[Storage] = None):
```

Comment:
What if this peer_did is already saved (probably with a different value)? I think we need to raise an error, as we assume only static peer dids right now.